build.gradle

```diff
@@ -62,11 +62,17 @@ java {
   withSourcesJar()
 }
 
+tasks.register('testJar', Jar) {
```

Comment:
Also providing a testJar artifact so upstream SDKs can use our handy test utilities. Sharing is caring!